server/automute.go

```diff
@@ -126,15 +126,15 @@ func (p *Plugin) setAutomuteIsEnabledForUser(userID string, channelsAutomuted bo
 	return nil
 }
 
-// func (p *Plugin) isUsersPrimaryPlatformTeams(userID string) bool {
-// 	pref, appErr := p.API.GetPreferenceForUser(userID, PreferenceCategoryPlugin, PreferenceNamePlatform)
-// 	if appErr != nil {
-// 		// GetPreferenceForUser returns an error when a preference is unset, so we default to MM being primary platform
-// 		return false
-// 	}
+func (p *Plugin) isUsersPrimaryPlatformTeams(userID string) bool {
+	pref, appErr := p.API.GetPreferenceForUser(userID, PreferenceCategoryPlugin, PreferenceNamePlatform)
+	if appErr != nil {
+		// GetPreferenceForUser returns an error when a preference is unset, so we default to MM being primary platform
```

Comment:
Can we distinguish an error from being unset from a different kind of error?

Reply:
Yeah, looks like we can actually. I thought that wouldn't work because `GetPreferenceForUser` returns an error which wraps `sql.ErrNoRows`, but we can actually test for that with `errors.Is(err, sql.ErrNoRows)`